app/controllers/api/v1x0/templates_controller.rb

```diff
@@ -1,24 +1,17 @@
 module Api
   module V1x0
     class TemplatesController < ApplicationController
-      include Api::V1x0::Mixins::IndexMixin
-      include Api::V1x0::Mixins::RBACMixin
-
-      before_action :read_access_check, :only => %i[show]
+      include Mixins::IndexMixin
 
       def show
         template = Template.find(params.require(:id))
+        authorize template
```

Comment:
@hsong-rh Should there be permissions for Template in the approval.json. I didn't see it in the rbac_config PR.
Is the template the same as workflow?

Reply:
We don have permission for template in approval.json. Only have `approval:templates:read` for approval admin